po/af.po

```diff
@@ -7,7 +7,7 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: MusicBrainz\n"
-"Report-Msgid-Bugs-To: http://tickets.musicbrainz.org/\n"
+"Report-Msgid-Bugs-To: https://tickets.metabrainz.org/\n"
```

Comment:
Not sure about this, we are pulling those files from transifex, the only way i'm aware of is to push .po files, with the (slight) risk to overwrite changes made through transifex.

In the mean time, can you split source code changes (.py) and translation file changes ?